service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderController.java

```diff
@@ -133,14 +137,21 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(
 
     @PostMapping("/about-to-submit")
     public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(
-        @RequestBody CallbackRequest callbackRequest) {
+        @RequestBody CallbackRequest callbackRequest,
+        @RequestHeader(value = "authorization") String authorization,
+        @RequestHeader(value = "user-id") String userId) throws IOException {
```

Comment:
please use RequestData object, you can actually refactor out all the request headers in the class if you don't think it's too much effort doing this

Reply:
done :)